src/test_reference.jl

```diff
@@ -85,9 +86,9 @@ end
 
 function test_reference(
     filename::AbstractString, raw_actual;
-    by = nothing, render = nothing, kw...)
+    by = nothing, render = nothing, force_raw_txt = false, kw...)
 
-    test_reference(query_extended(filename), raw_actual, by, render; kw...)
+    test_reference(query_extended(filename, force_raw_txt), raw_actual, by, render; kw...)
 end
```

Comment:
Instead of exposing a keyword `force_raw_txt`, it's probably better to directly expose a positional argument `format` to function `test_reference` (and keyword argument `format=nothing` to `@test_reference`)

```suggestion
function test_reference(
    filename::AbstractString, raw_actual;
    format=nothing, by = nothing, render = nothing)

    format = isnothing(format) ? query_extended(filename) : format
    format isa AbstractString && (format = FileIO.DataFormat{Symbol(format)})
    test_reference(format, filename, raw_actual, by, render; kw...)
end
```

So that users can instead force a loading format (not only string data):

```julia
@test_reference "reference_file.txt" actual format="TXT"
@test_reference "reference_file.txt" actual format=FileIO.format"TXT"
```